src/main/java/uk/gov/companieshouse/certifiedcopies/orders/api/interceptor/UserAuthorisationInterceptor.java

```diff
@@ -0,0 +1,108 @@
+package uk.gov.companieshouse.certifiedcopies.orders.api.interceptor;
+
+import org.springframework.web.servlet.HandlerMapping;
+import org.springframework.web.servlet.handler.HandlerInterceptorAdapter;
+import uk.gov.companieshouse.certifiedcopies.orders.api.model.CertifiedCopyItem;
+import uk.gov.companieshouse.certifiedcopies.orders.api.service.CertifiedCopyItemService;
+import uk.gov.companieshouse.certifiedcopies.orders.api.util.EricHeaderHelper;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+import static org.springframework.http.HttpMethod.GET;
+import static org.springframework.http.HttpMethod.POST;
+import static org.springframework.http.HttpStatus.NOT_FOUND;
+import static org.springframework.http.HttpStatus.UNAUTHORIZED;
+import static uk.gov.companieshouse.certifiedcopies.orders.api.logging.LoggingUtils.*;
+
+public class UserAuthorisationInterceptor extends HandlerInterceptorAdapter {
+
+    private final CertifiedCopyItemService service;
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(APPLICATION_NAMESPACE);
```

Comment:
Logger should be obtained and used from the LoggintUtils class - see CertifiedCopiesItemController for an example of the usage